README.md

```diff
@@ -40,14 +40,14 @@ plug(
 ```
 config :torch,
   otp_app: :my_app_name,
-  template_format: "eex" || "slim"
```

Comment:
I like updating the README here to show the new expected option value of "slime".  I think we still need to support the value of "slim" as well for backwards compatibility.

Later in our tasks where we check the `template_format` value, we should coerce `"slim"` into `"slime"`